.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    groups:
+      dependencies:
+        patterns:
+          - "*"
+  - package-ecosystem: "swift"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 6
+    allow:
+      - dependency-type: all
+    groups:
+      all-dependencies:
+        patterns:
+          - "*"
```

Comment:
We don't commit `Package.resolved` in this repo since it's just a library, so it there a point?

```suggestion
```

Reply:
Yes.

Reply:
What then? 😶‍🌫️

Reply:
Ya know what? Anything to make you happy, you bloody annoying person who's learned far too well from me and would be impressing me with the nitpickiness if I weren't being deliberately over the top ornery 😉